corehq/ex-submodules/couchforms/tests/test_analytics.py

```diff
@@ -179,31 +123,93 @@ def test_get_exports_by_form(self):
 class CouchformsESAnalyticsTest(TestCase):
     domain = 'hqadmin-es-accessor'
 
-    def setUp(self):
-        super(CouchformsESAnalyticsTest, self).setUp()
+    @classmethod
+    def setUpClass(cls):
+        @patch('couchforms.analytics.FormES.index', XFORM_INDEX_INFO.index)
+        @patch('corehq.apps.es.es_query.ES_META', TEST_ES_META)
+        @patch('corehq.elastic.ES_META', TEST_ES_META)
+        def create_form_and_sync_to_es(received_on):
+            with process_kafka_changes('XFormToElasticsearchPillow'):
+                with process_couch_changes('DefaultChangeFeedPillow'):
+                    metadata = TestFormMetadata(domain=cls.domain, app_id=cls.app_id,
+                                                xmlns=cls.xmlns, received_on=received_on)
+                    form = get_form_ready_to_save(metadata, is_db_test=True)
+                    form_processor = FormProcessorInterface(domain=cls.domain)
+                    form_processor.save_processed_models([form])
+
+        from casexml.apps.case.tests.util import delete_all_xforms
+        delete_all_xforms()
+        cls.now = datetime.datetime.utcnow()
+        cls._60_days = datetime.timedelta(days=60)
+        cls.domain = 'my_crazy_analytics_domain'
+        cls.app_id = uuid.uuid4().hex
+        cls.xmlns = 'my://crazy.xmlns/'
         with trap_extra_setup(ConnectionError):
-            self.elasticsearch = get_es_new()
-            initialize_index_and_mapping(self.elasticsearch, XFORM_INDEX_INFO)
+            cls.elasticsearch = get_es_new()
+            initialize_index_and_mapping(cls.elasticsearch, XFORM_INDEX_INFO)
+            cls.forms = [create_form_and_sync_to_es(cls.now), create_form_and_sync_to_es(cls.now-cls._60_days)]
```

Comment:
I think the `create_form_and_sync_to_es` method doesn't return anything so this becomes a list of `None`s. You aren't using the contents of this anywhere, but might be worth changing for whoever comes next.